Content.Server/Lathe/LatheSystem.cs

```diff
@@ -243,8 +250,8 @@ private void UserInterfaceOnOnReceiveMessage(EntityUid uid, LatheComponent compo
                             component.Queue.Enqueue(recipe);
                             component.UserInterface?.SendMessage(new LatheFullQueueMessage(GetIdQueue(component)));
                         }
-                        if (!HasComp<LatheProducingComponent>(component.Owner) && component.Queue.Count > 0)
-                            Produce(component, component.Queue.Dequeue());
+                    if (!HasComp<LatheProducingComponent>(component.Owner) && component.Queue.Count > 0)
+                        Produce(component, component.Queue.Dequeue());
```

Comment:
why was indentation changed here?

Reply:
Ah must have been an accident

Reply:
Oh I think I clicked the "Fix all indentation" and it automaticly fixed the indentation